### DISCUSSION TITLE
Github Pages only effecting README.md

### DISCUSSION THREAD

**Discussion by Ferynix** [community/community#23013] [Other Feature Feedback, Questions, & Ideas]
<p>So, I have README.md and MergingAssemblies.md</p>
<p>my yml file is enabling the cayman jekyll theme, but the theme is only effecting the README file.</p>
<p>Why is that, and how do I make it global to effect all the markdown files?</p>
<p>I plan on making MUCH MORE markdown files in my tutorials repo: <a href="https://github.com/skylerspark/Tutorials" rel="nofollow noopener">https://github.com/skylerspark/Tutorials</a></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/skylerspark">@skylerspark</a>,</p>
<p>Thank you for being here, looks like you are linking directly to the <code>.md</code> file?  <span>If you drop the </span><code>.md</code><span>it works (e.g. </span><a href="https://skylerspark.github.io/Tutorials/MergingAssemblies" target="_blank" rel="nofollow noopener">https://skylerspark.github.io/Tutorials/MergingAssemblies</a><span> or </span><a href="https://skylerspark.github.io/Tutorials/MergingAssemblies.html" target="_blank" rel="nofollow noopener">https://skylerspark.github.io/Tutorials/MergingAssemblies.html)</a></p>
<p>Could you try this and let us know if it works. I hope this helps!</p>

**Comment by Ferynix** [comment]
<p>Thankyou, this solved my issue :slight_smile:</p>
